src/Core/package.hpp

```diff
@@ -14,6 +14,7 @@ namespace ShadyCore {
 	protected:
 		class Key {
 		public:
+			const std::string actualName;
```

Comment:
I don't think this here is good, Key is optimized for hashing and try to avoid copies, since it is used a lot in critical parts of the code.

Reply:
actually, the copy and move is deleted, so it should be ok, but i think that there's better ways to handle. I will check later